libs/serialization/tests/unit/serialization_boost_variant.cpp

```diff
@@ -93,6 +102,7 @@ int main()
 
     HPX_TEST_EQ(sivar.which(), sovar.which());
     HPX_TEST_EQ(sivar, sovar);
+#endif
 
     return hpx::util::report_errors();
```

Comment:
`#include <hpx/testing.hpp>` needs to be outside the `#ifdef` at the top for this, or this should just be `return 0` if `HPX_SERIALIZATION_HAVE_BOOST_TYPES` isn't defined (https://cdash.cscs.ch//viewBuildError.php?buildid=98967).